client/my-sites/plans/ecommerce-trial/index.tsx

```diff
@@ -0,0 +1,320 @@
+				},
+				{
+					title: 'Cart and checkout optimization',
+					subtitle:
+						'Streamline your checkout and boost conversions with the Cart and Checkout blocks.',
+				},
+			],
+		},
+		{
+			illustration: growth,
+			title: 'Marketing and growth',
+			subtitle: 'Optimize your store for sales by adding in email and social integrations.',
+			items: [
+				{
+					title: 'Sell everywhere',
+					subtitle:
+						'Promote and sell your products on popular social media platforms and marketplaces.',
+				},
+				{
+					title: 'Automate your marketing',
+					subtitle: 'Build custom email automations to keep customers and prospects engaged.',
+				},
+				{
+					title: 'Recover abandoned carts',
+					subtitle:
+						'Drive more sales by automatically emailing customers who leave your store without checking out.',
+				},
+				{
+					title: 'Encourage referrals',
+					subtitle: 'Offer free gifts or coupons when your customers refer new shoppers.',
+				},
+				{
+					title: 'Send birthday coupons',
+					subtitle:
+						'Automatically email your customers a birthday discount to keep them coming back.',
+				},
+				{
+					title: 'Drive loyalty',
+					subtitle: 'Keep your loyal customers loyal with a rewards program.',
+				},
+			],
+		},
+		{
+			illustration: shipping,
+			title: 'Shipping',
+			subtitle: 'Streamline your fulfillment with integrated shipping.',
+			items: [
+				{
+					title: 'Print shipping labels',
+					subtitle: 'Print shipping labels from your store to save time and money.',
+				},
+				{
+					title: 'Offer shipment tracking',
+					subtitle: 'Provide customers with an easy way to track their shipment.',
+				},
+				{
+					title: 'Customize shipping rates',
+					subtitle:
+						'Define multiple shipping rates based on location, price, weight, or other criteria.',
+				},
+				{
+					title: 'Set conditional shipping',
+					subtitle: 'Restrict shipping and payment options using conditional logic.',
+				},
+				{
+					title: 'Simplify returns and exchanges',
+					subtitle:
+						'Manage the RMA process, add warranties to products and let customers request/manage returns from their account.',
+				},
+			],
+		},
+	];
+
+	const priceContent = isAnnualSubscription
+		? translate(
+				'{{monthlyPriceWrapper}}{{currencySymbol}}%(currency)s{{/currencySymbol}}%(monthlyPrice)s{{/monthlyPriceWrapper}} {{priceDescription}}per month, %(currency)s%(annualPrice)s billed annually{{/priceDescription}}',
+				{
+					args: {
+						currency: currencySymbol,
+						monthlyPrice: eCommercePlanPrices.annualPlanMonthlyPrice,
+						annualPrice: eCommercePlanPrices.annualPlanPrice,
+					},
+					components: {
+						monthlyPriceWrapper: <span className="e-commerce-trial-plans__price-card-value" />,
+						priceDescription: <span className="e-commerce-trial-plans__price-card-interval" />,
+						currencySymbol: <span className="e-commerce-trial-plans__price-card-value-symbol" />,
```

Comment:
Minor note: we could use dedicated `const`s for these components, as they are the same across the two logic branches, and it may be simpler to define them once and use them twice.
```suggestion
						monthlyPriceWrapper,
						priceDescription,
						currencySymbol,
```